server/services/expectedArrivalsService.ts

```diff
@@ -56,6 +57,15 @@ export default class ExpectedArrivalsService {
     return this.welcomeClientFactory(token).getArrival(id)
   }
 
+  public async getPrisonerDetailsForArrival(id: string): Promise<PrisonerDetails> {
+    const token = await this.hmppsAuthClient.getSystemClientToken()
+    const arrival = await this.welcomeClientFactory(token).getArrival(id)
+    if (arrival.potentialMatches.length > 1) {
+      logger.warn(`multiple matches for move: ${id}`)
+    }
+    return arrival.potentialMatches[0]
+  }
```

Comment:
this provides the actual prisoner details from nomis associated with the arrival.

This assumes there is only a single potential match